src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonMigrationService.kt

```diff
@@ -275,7 +274,8 @@ fun ContactPerson.toDpsMigrateContactRequest(): MigrateContactRequest = MigrateC
           createUsername = restriction.audit.createUsername,
           modifyDateTime = restriction.audit.modifyDatetime.toDateTime(),
           modifyUsername = restriction.audit.modifyUserId,
-          staffUsername = restriction.enteredStaff.username,
+          // DPS Might put this back in again
+          // staffUsername = restriction.enteredStaff.username,
```

Comment:
in discussions with DPS - I think they still need this but they have removed from their DTO